devfiles/java-springboot/devfile.yaml

```diff
@@ -1,52 +1,44 @@
 schemaVersion: 2.0.0
 metadata:
   name: java-springboot
-  version: 1.0.0
+  version: 1.1.0
 starterProjects:
   - name: springbootproject
     git:
       location: "https://github.com/odo-devfiles/springboot-ex.git"
```

Comment:
This needs to be updated. Devfile spec does not allow git.location. See Tomas' PR https://github.com/odo-devfiles/registry/pull/42 and devfile schema https://github.com/devfile/api/blob/master/schemas/devfile.json

Reply:
```
Maysuns-MacBook-Pro:springboot-ex maysun$ odo push --show-log
 ✗  invalid devfile schema. errors :
- starterProjects.0.git: Additional property location is not allowed
```

Reply:
Done